plugin-ozone/src/main/java/org/apache/ranger/authorization/ozone/authorizer/RangerOzoneAuthorizer.java

```diff
@@ -151,6 +160,18 @@ public boolean checkAccess(IOzoneObj ozoneObject, RequestContext context) {
 		}
 
 		try {
+			if (snapShotName != null) {
+				rangerResource.setValue(KEY_RESOURCE_KEY,".snapshot/" + snapShotName + "/" + ozoneObj.getKeyName());
+				RangerAccessResult result = rangerPlugin
+						.isAccessAllowed(rangerRequest);
+				if (result == null) {
+					LOG.error("Ranger Plugin returned null. Returning false");
+					return false;
```

Comment:
If we return false from here for snapshotted path, we will miss out on emitting logs. Check the corresponding code path for non-snapshotted path below.